advocacy_docs/supported-open-source/patroni/debian11_quick_start.mdx

```diff
@@ -167,7 +171,7 @@ EOF
 
 `$MY_IP` and `$MY_NAME` are specific to the local host. Otherwise, the `patroni.yml` configuration is the same on all Patroni nodes.
 
-Patroni expects to find the `postgres` binary in the `bin_dir` location, while the EDB Postgres Advanced Server binary is called `edb-postgres`. Create a symbolic link to let Patroni find that binary:
+Patroni expects to find the `postgres` binary in the `bin_dir` location, while the EDB Postgres Advanced Server binary is called `edb-postgres`. As the `postgresql.bin_name` setting was introduced in the [`3.0.3` release](https://github.com/zalando/patroni/blob/master/docs/releases.rst#version-303), remove it from the configuration if you are running an older version and create a symbolic link to let Patroni find that binary:
```

Comment:
Specify that it's an older version of Patroni:
"... if you are running an older version of Patroni and create..."

Reply:
The order might be what confuses me there.
_The `postgresql.bin_name` setting was introduced in the [`3.0.3` release], so if you are running an older version of Patroni remove it from the configuration, and create a symbolic link to let Patroni find the server's binary:_
Still two Patroni's but a bit more sparse

Reply:
Suggestion.

The `postgresql.bin_name` setting does not exist in Patroni prior to the [release of version 3.0.3]((https://github.com/zalando/patroni/blob/master/docs/releases.rst#version-303) and will be silently ignored by older versions. For these versions, use an appropriately named symbolic link in the `bin_dir` directory that points to the relevant Postgresql binary.

Reply:
I went for:

The `postgresql.bin_name` setting does not exist in Patroni prior to the [`3.0.3` release](https://github.com/zalando/patroni/blob/master/docs/releases.rst#version-303) and will be silently ignored by older versions. For these versions, create an appropriately named symbolic link that points to the relevant binary